### PR TITLE
Fix suitesparse path

### PR DESCRIPTION
**Pull request by appolloford**
(created using `eb --new-pr`)

Fix the issue from suitesparse easyblock update: https://github.com/easybuilders/easybuild-easyblocks/issues/3051

### CODE REVIEW

File: easybuild/easyconfigs/c/CVXOPT/CVXOPT-1.3.1-foss-2022a.eb [hunk 1]
```diff
@@ -32,8 +32,14 @@ use_pip = True
 sanity_pip_check = True
 download_dep_fail = True
 
-preinstallopts = 'CVXOPT_BUILD_FFTW=1 CVXOPT_BUILD_GSL=1 CVXOPT_BLAS_EXTRA_LINK_ARGS="$LIBLAPACK" '
-preinstallopts += 'CVXOPT_FFTW_EXTRA_LINK_ARGS="$LIBFFT" CVXOPT_SUITESPARSE_SRC_DIR=$EBROOTSUITESPARSE'
+preinstallopts = " ".join([
+    'CVXOPT_BUILD_FFTW=1',
+    'CVXOPT_BUILD_GSL=1',
+    'CVXOPT_BLAS_EXTRA_LINK_ARGS="$LIBLAPACK"',
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/CVXOPT/CVXOPT-1.3.1-foss-2022a.eb:38]: Does CVXOPT need Lapack or just BLAS?
In either case using just LIBLAPACK is wrong.
If it need both then it needs $LIBLAPACK $LIBBLAS (they may not be the same)
and if it just needs BLAS then it should be just $LIBBLAS

Comment (appolloford) [reply easybuild/easyconfigs/c/CVXOPT/CVXOPT-1.3.1-foss-2022a.eb]: It seems that both BLAS and LAPACK are required. I fix the flags now.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/fab12e956572c882316bb92b99ce8af4 for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Comment by Micket** [comment]
Going in, thanks @appolloford!

**Pull request merged**